### ISSUE TITLE
splitfasta wrapper

### ISSUE THREAD

**Issue by mmiladi** [bgruening/galaxytools#507]
Hi @bgruening ,

@TorHou and I are trying to split a large fasta file using splitFasta wrapper.
Inside FR-Galaxy the splitting of the fasta file is shown as still ongoing, even though the job is finished on the server. 

Could you please have a look to the xml, the conditional parameter `splitmode` is not available in FR-Galaxy instance, I guess there might be an inconsistency. Also there is a chance to be an IO issue or BioPython inefficiency 
https://github.com/bgruening/galaxytools/blob/3598a6be740f49d349fc72d3447d8e9b612981ed/tools/splitfasta/splitFasta.xml#L8

**Comment by bgruening** [comment]
Splitmode is the conditional below in the inputs, isn't it? Does it work for small files?

**Comment by mmiladi** [comment]
No I cannot see the conditional in my browser.

**Comment by mmiladi** [comment]
Yup it works for small files. Is it about splitting a big fasta file to single files in Galaxy (or FR-Galaxy..) not a good practice?

**Comment by TorHou** [comment]
We should update from 0.2.0 to 0.3.0 of this tools on our server ..
